ibm/mas_devops/roles/suite_db2_setup_for_manage/tasks/main.yml

```diff
@@ -77,13 +77,20 @@
 - name: Run setupdb to prepare Db2 for use with Maximo TPAE
   shell: oc exec -n {{db2_namespace}} {{db2_pod_name}} -- su -lc '/tmp/setupdb.sh | tee /tmp/setupdb.log' db2inst1
   register: prepare_cmds_status
+  retries: 10
+  delay: 30 # seconds
+  until: prepare_cmds_status.rc == 0
 
 - name: "Database setup debug information"
   debug:
     msg:
       - "Result (stdout) ........................ {{ prepare_cmds_status.stdout }}"
       - "result (stderr) ........................ {{ prepare_cmds_status.stderr }}"
 
+- fail: msg="setupdb.sh failed for Manage Maxinst -  tablespace or bufferpools may not have been created)"
```

Comment:
@sachbalag suggestion here is to do the assertion like it is done in other roles, i.g. https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/uds/tasks/install/determine-storage-classes.yml#L28 - just to keep the adopted standard/convention used everywhere else